src/main/java/sx/blah/discord/util/MessageBuilder.java

```diff
@@ -207,12 +224,13 @@ public boolean isUsingTTS() {
 	 * @throws RateLimitException
 	 * @throws DiscordException
 	 * @throws MissingPermissionsException
+	 * @throws FileNotFoundException
 	 */
-	public IMessage build() throws DiscordException, RateLimitException, MissingPermissionsException {
+	public IMessage build() throws DiscordException, RateLimitException, MissingPermissionsException, FileNotFoundException {
```

Comment:
FileNotFoundException is not a RuntimeException which will lead to problems with current code.